MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -57,31 +57,69 @@ The [DisplayMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.Cha
 
 {% highlight xaml %}
 
-<chart:SfCartesianChart>
-  ...
- <chart:SfCartesianChart.TrackballBehavior>
-    <chart:ChartTrackballBehavior ShowLine="True" 
-                                  DisplayMode="NearestPoint"/>
- </chart:SfCartesianChart.TrackballBehavior>
-     ...
-</chart:SfCartesianChart>
+    <chart:SfCartesianChart>
+      ...
+    <chart:SfCartesianChart.TrackballBehavior>
+        <chart:ChartTrackballBehavior ShowLine="True" 
+                                      DisplayMode="NearestPoint"/>
+    </chart:SfCartesianChart.TrackballBehavior>
+        ...
+    </chart:SfCartesianChart>
 
 {% endhighlight %}
 
 {% highlight c# %}
 
- SfCartesianChart chart = new SfCartesianChart();
- ...
- ChartTrackballBehavior trackball = new ChartTrackballBehavior();
- trackball.ShowLine = true;
- trackball.DisplayMode = LabelDisplayMode.NearestPoint;
- . . . 
+    SfCartesianChart chart = new SfCartesianChart();
+    ...
+    ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+    trackball.ShowLine = true;
+    trackball.DisplayMode = LabelDisplayMode.NearestPoint;
+    . . . 
 
 {% endhighlight %}
 
 {% endtabs %}
 
-## Apperance customization
+## Activation mode
+
+The [ActivationMode]() property is used to restrict the visibility of trackball based on the touch actions. The default value of this property is `ChartTrackballActivationMode.LongPress`.
+
+The ChartTrackballActivationMode enum contains the following values:
+
+* `LongPress` – Activates trackball only when performing the long press action.
+* `TouchMove` – Activates trackball only when performing touch move action.
+* `None` – Hides the visibility of trackball when setting activation mode to `None`. It will be activated when calling 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+      ...
+    <chart:SfCartesianChart.TrackballBehavior>
+        <chart:ChartTrackballBehavior ActivationMode = "LongPress"/>
```

Comment:
@Harsha-SF4223  ,As its default value is "LongPress,"  why do we need to explain the code snippet?

Reply:
Based on platform, the default values changes. 
In Windows and Mac, the default value is TouchMove.
In Android and iOS, the default value is LongPress.
So, irrespective of the platform, I have assigned the value for activation mode as LongPress.